homeworks/vladislav.buzunar_buzyaka/4-objects/objects.js

```diff
@@ -0,0 +1,44 @@
+const tamagochi = {
+    name: 'Tamagochi',
+    age: 23,
+    healthLevels: 5,
+    happyLevels: 3,
+    foodLevels: 4,
+    noEating() {
+        tamagochi.healthLevels -= 1;
```

Comment:
It would be best if you did not use the name of the object (`tamagochi`). Use `this` instead for all calls